### ISSUE TITLE
Selection after Yank Command Issue

### ISSUE THREAD

**Issue by ghachey** [zbrad/EmacsKeys#6]
Hi,

Started using this extension and so far got so good, it's got several of the basic editing features which is all I was asking for. One things that is very annoying though, not sure if it is a bug or expected behavior, but this is not default behavior of recent GNU Emacs.

After a Ctrl-Y (Yank or Paste) the cursor is in highlight mode and highlights text starting from the end of the yank location. I have to cancel with Crtl-G to remove this highlighting.

**Comment by ywabiko** [comment]
+1
However I could not find a good VS API to do this (disable highlight with region enabled)

**Comment by zbrad** [comment]
yeah, I haven't tracked it down, and it bugs me as well.   So far I just ^G it.
